client/landing/subscriptions/components/site-subscription-page/site-subscription-page.tsx

```diff
@@ -38,6 +40,20 @@ const SiteSubscriptionPage = () => {
 		deliveryFrequency,
 		subscribers,
 	} = data;
+	const [ showSettings, setShowSettings ] = useState( true );
```

Comment:
Do we anticipate the settings not to be displayed in some other cases, then unsubscribed? 🤔

Reply:
The reason I kept the `showSettings` state there (instead of relying on `unsubscribed` - as with the Success notice) was to have an easy way to display the settings again when the upcoming "Resubscribe" button is clicked and the user is subscribed again. 🤔 

Do you think that it should be better to simply rely on `unsubscribed`? I just felt that I don't have full control over its value, but I might be overthinking thia.

Reply:
To clarify a bit more: Let's imagine that the mutation is successful and hence `unsubscribed` is set to `true`. Then the user clicks the "Resubscribe" link (not implemented yet), but the `unsubscribed` hasn't changed. As a result, the settings would be still hidden.